.travis.yml

```diff
@@ -18,6 +18,7 @@ matrix:
     - php: hhvm
 
 before_install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi
```

Comment:
@xabbuh yes ... let me ask differently: why not remove `xdebug.ini` always? After all, HHVM doesn't have xdebug, so the result would be the same.